packages/react/src/HeroImage/HeroImage.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import clsx from 'clsx'
+import { forwardRef, type ImgHTMLAttributes, type PropsWithChildren, type ReactNode } from 'react'
+import { Image } from '../Image/Image'
+
+export interface HeroImageProps extends PropsWithChildren<ImgHTMLAttributes<HTMLImageElement>> {
+  /*
+   * A custom image component to use instead of the one from this library.
+   * Note that the image must fill the entire content box, e.g. through `object-fit: cover`.
+   */
+  ImageComponent?: ReactNode
+}
+
+export const HeroImage = forwardRef<HTMLImageElement, HeroImageProps>(
+  ({ children, className, ImageComponent, ...restProps }: HeroImageProps, ref) => (
+    <div className={clsx('amsterdam-hero-image', className)}>
+      <div className="amsterdam-hero-image__image">
+        {ImageComponent ?? <Image {...restProps} alt="" objectFit="cover" ref={ref} />}
```

Comment:
Do we want to render `<Image />` here at all? We could just say that HeroImage is a container with slots where other components are rendered. We might not even need `amsterdam-hero-image__image`, if you can just pass `<AspectRatio><Image /></AspectRatio>` in the first slot.

Maybe slots aren't even necessary, if HeroImage just renders its children :D All HeroImage would do is just render its children over one another.

Reply:
It is with a heavy heart that I have stripped this component completely to a General Thing that assumes nothing and stacks its children 🥲

We can use the name `Stack`, which some other libraries use for one-dimensional layout components, as we will separate our future `Row` and `Column` components.

Reply:
Discussed and decided to go with ‘Overlap’.